cli/azd/pkg/project/service_target_springapp.go

```diff
@@ -200,10 +199,8 @@ func (st *springAppTarget) Endpoints(
 		return nil, fmt.Errorf("fetching service properties: %w", err)
 	}
 
-	endpoints := make([]string, len(springAppProperties.Fqdn))
-	for idx, fqdn := range springAppProperties.Fqdn {
-		endpoints[idx] = fmt.Sprintf("https://%s/", st.extractEndpoint(fqdn, serviceConfig.Name))
-	}
+	endpoints := make([]string, len(springAppProperties.Url))
+	copy(endpoints, springAppProperties.Url)
 
 	return endpoints, nil
```

Comment:
We're fine to just return the `[]string` directly. 

```suggestion
	return springAppProperties.Url, nil
```